.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+#### Motivation
```

Comment:
I'll include the descriptions for each section, but I prefer the Markdown section header. That way extra "hard coded" newlines aren't necessary ... the spacing is determined by the styles.